app/Http/Controllers/User/AdminOrderInvoiceController.php

```diff
@@ -138,14 +170,10 @@ public function getPaymentDetail($id)
                             value=".$model->id.' name=select[] id=checkpayment>';
                         })
                         ->addColumn('invoice_no', function ($model) {
-                            return $model->invoice()->first() ? $model->invoice()->first()->number : '--';
+                            return $model->invoice()->first() ? '<a href='.url('invoices/show?invoiceid='.$model->invoice()->first()->id).'>'.$model->invoice()->first()->number.'</a><br>' : '--';
```

Comment:
use `->count()` instead. It queries all the data from the database just for checking its existence.

Reply:
Fixed